final_prep/mod_zero_hero.rb

```diff
@@ -1,41 +1,74 @@
 # Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
+hero_name = "Batman"
+special_ability = "Wealth"
+
 
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
+greeting = "I'm #{hero_name}!"
+catchphrase = "I will solve your problems using my #{special_ability}!"
+
+
 
 # Declare two variables - power AND energy - set to integers
+power = 999
+energy = 9000
 
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
+full_power = power * 500
+full_energy = energy + 150
 
-# Declare two variables - is_human and identity_concealed - assigned to booleans
 
+# Declare two variables - is_human and identity_concealed - assigned to booleans
+is_human = true
+identity_concealed = true
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
+arch_enemies = ["The Joker", "The Penguin", "Mr. Freeze"]
+sidekicks = ["Robin", "Alfred", "Superman"]
 
-# Print the first sidekick to your terminal
 
+# Print the first sidekick to your terminal
+puts sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+puts arch_enemies[2]
 # Write some code to add a new arch_enemy to the arch_enemies array
-
+arch_enemies.push("Bane")
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
-
+puts  arch_enemies[3]
 # Remove the first sidekick from the sidekicks array
-
+sidekicks.delete_at(0)
 # Print the sidekicks array to terminal to ensure you added a new sidekick
-
+puts sidekicks
 # Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
 
+
+def assess_situation(danger_level, save_the_day, bad_excuse)
+  if danger_level <= 10
+    puts "Meh. Hard Pass"
+    elsif danger_level > 50
+    puts "#{bad_excuse}"
+  else danger_level > 10
```

Comment:
Here you can either use else, which will be run if none of the other conditionals or met or you can use _elsif danger_level > 10 &&  danger_level < 50_